src/Schema/ComparatorConfig.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Schema;
+
+class ComparatorConfig
```

Comment:
Let's lock this down as much as possible until asked to open it? `final`, `readonly`, `private`?

Reply:
Done in 9180776e1161220c18b1e28422a9d3bbcd62a59e